README.md

```diff
@@ -14,35 +15,34 @@ See benchmark results [here](https://intelpython.github.io/scikit-learn_bench).
 * [Running Python benchmarks with runner script](#running-python-benchmarks-with-runner-script)
 * [Supported algorithms](#supported-algorithms)
 * [Algorithms parameters](#algorithms-parameters)
-* [Legacy automatic building and running](#legacy-automatic-building-and-running)
 
 ## Prerequisites
 - `python` and `scikit-learn` to run python versions
 - pandas when using its DataFrame as input data format
-- `icc`, `ifort`, `mkl`, `daal` to compile and run native benchmarks
 - machine learning frameworks, that you want to test. Check [this item](#how-to-create-conda-environment-for-benchmarking) to get additional information how to set environment.
 
 ## How to create conda environment for benchmarking
 
 Create a suitable conda environment for each framework to test. Each item in the list below links to instructions to create an appropriate conda environment for the framework.
 
-* [**scikit-learn**](https://github.com/PivovarA/scikit-learn_bench/blob/master/sklearn/README.md#how-to-create-conda-environment-for-benchmarking)
-* [**daal4py**](https://github.com/PivovarA/scikit-learn_bench/blob/master/daal4py/README.md#how-to-create-conda-environment-for-benchmarking)
-* [**cuml**](https://github.com/PivovarA/scikit-learn_bench/blob/master/cuml/README.md#how-to-create-conda-environment-for-benchmarking)
-* [**xgboost**](https://github.com/PivovarA/scikit-learn_bench/tree/master/xgboost/README.md#how-to-create-conda-environment-for-benchmarking)
+* [**scikit-learn**](https://github.com/IntelPython/scikit-learn_bench/blob/master/sklearn/README.md#how-to-create-conda-environment-for-benchmarking)
+* [**daal4py**](https://github.com/IntelPython/scikit-learn_bench/blob/master/daal4py/README.md#how-to-create-conda-environment-for-benchmarking)
+* [**cuml**](https://github.com/IntelPython/scikit-learn_bench/blob/master/cuml/README.md#how-to-create-conda-environment-for-benchmarking)
+* [**xgboost**](https://github.com/IntelPython/scikit-learn_bench/tree/master/xgboost/README.md#how-to-create-conda-environment-for-benchmarking)
 
 ## How to enable daal4py patching for scikit-learn benchmarks
 Set specific environment variable `export FORCE_DAAL4PY_SKLEARN=YES`
 
 ## Running Python benchmarks with runner script
 
-Run `python runner.py --configs configs/config_example.json [--output-format json --verbose]` to launch benchmarks.
+Run `python runner.py --configs configs/config_example.json [--output-file result.json --verbose INFO --report]` to launch benchmarks.
 
 runner options:
 * ``configs`` : configuration files paths
 * ``dummy-run`` : run configuration parser and datasets generation without benchmarks running
-* ``verbose`` : print additional information during benchmarks running
-* ``output-format``: *json* or *csv*. Output type of benchmarks to use with their runner
+* ``verbose`` : *WARNING*, *INFO*, *DEBUG*. Print additional information during benchmarks running. Default is *INFO*
```

Comment:
Is this a standard way to specify verbose information levels? It is not clear to me how these levels are related to each other.

Reply:
Right, it is standard verbose . But I agree to add more information for each level

Reply:
Add info